recipes-mender/mender/files/mender.service

```diff
@@ -6,6 +6,7 @@ Type=simple
 User=root
 Group=root
 ExecStartPre=/bin/mkdir -p -m 0700 /data/mender
+ExecStartPre=/bin/ln -s /etc/mender/tenant.conf /var/lib/mender/authtentoken
```

Comment:
The quote I gave mentions the prefix "-", which seems to be used to silence such failures, at least the way I read it. But using `-sf` also works.